features/bevragen/zoek-fout-cases.feature

```diff
@@ -106,17 +106,39 @@ Rule: Er moet een valide zoek type worden opgegeven
     | status   | 406                                                         |
     | instance | /haalcentraal/api/brp/personen                              |
 
-  Scenario: */* als Accept contenttype wordt ondersteund
+  Abstract Scenario: '<accept media type>' als Accept contenttype wordt ondersteund
     Als personen wordt gezocht met de volgende parameters
     | naam                | waarde                          |
     | type                | RaadpleegMetBurgerservicenummer |
     | burgerservicenummer | 000000024                       |
     | fields              | burgerservicenummer             |
-    | header: Accept      | */*                             |
+    | header: Accept      | <accept media type>             |
     Dan heeft de response 0 personen
 
+    Voorbeelden:
+    | accept media type               |
+    | */*                             |
+    | application/json                |
+    | application/json; charset=utf-8 |
+    | application/json;charset=utf-8  |
+
```

Comment:
en */*; charset=utf-8? moet dat ook worden ondersteund?

Reply:
of bedoel je deze als juist tegenover de fout cases om te tonen wat net geen fout geeft?

Reply:
dat kan ik ook toevoegen

Reply:
@MelvLee ga je deze nog toevoegen? Als je daar nu geen tijd voor hebt ook best. Wat mij betreft is de PR voor de rest in ieder geval approved

Reply:
toegevoegd